### ISSUE TITLE
Joining OASIS Committees - broken link

### ISSUE THREAD

**Issue by mbminto** [oasis-dev/oasis-new-website#141]
On the [My OASIS Homepage](https://www.oasis-open.org/kws/), down in the section called Important Links -> Technical Work, there's a broken link called "[Joining OASIS Committees](https://www.oasis-open.org/committees/join.php)."

**Comment by OASIS-OP-Admin** [comment]
I don't think that I have access to fix that link. I will check with Jesse/Bitlancer.
